packages/bridge/src/nitro/index.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  NativeEVMTokenConstantAddress,
+  TronChainInfo,
+} from "@osmosis-labs/utils";
+import basex from "base-x";
+
+import {
+  BridgeAsset,
+  BridgeChain,
+  BridgeExternalUrl,
+  BridgeProvider,
+  BridgeProviderContext,
+  BridgeQuote,
+  BridgeTransactionRequest,
+  GetBridgeExternalUrlParams,
+  GetBridgeSupportedAssetsParams,
+} from "../interface";
+
+const BASE58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
+const bs58 = basex(BASE58);
+
+export class NitroBridgeProvider implements BridgeProvider {
+  static readonly ID = "Nitro";
+  readonly providerName = NitroBridgeProvider.ID;
+
+  constructor(protected readonly ctx: BridgeProviderContext) {}
+
+  async getQuote(): Promise<BridgeQuote> {
+    throw new Error("Nitro quotes are currently not supported.");
+  }
+
+  async getSupportedAssets({
```

Comment:
Should we catch and warn and return [] incase anything unexpected happens? Or, will client just tolerate that